### DISCUSSION TITLE
Xcode imageLiteral

### DISCUSSION THREAD

**Discussion by dirtynapk1ns** [SwifterSwift/SwifterSwift#1040] [Q&A]
Does anyone know how to get the imageLiteral command to in Xcode when working with swift. Got The Who.what = part down but Xcode wants nothing to do with imageLiteral

**Comment by maheenqaisar** [comment]
To use the `imageLiteral` command in Xcode when working with Swift, you will first need to add the image to your project. To do this, go to the Assets.xcassets folder, click the + icon, and select the image you want to use. Once you have done this, you can use the `imageLiteral` command to reference the image.
